### PR TITLE
Add EIP: Buy Russia for free gas

### PR DESCRIPTION
**Pull request by backmeupplz**
Russia has a lot of gas. Also the Russia-Ukraine war. Gas prices are too damn high. I had to pay $60 for Gnosis Safe transaction yesterday. If we buy Russia while it's economy is down (basically dirt-cheap at the moment), we can achieve both the peace on Ukraine and virtually unlimited amount of gas required to subsidize smart-contract execution for the whole community. It's a win-win.

### CODE REVIEW

**Comment by backmeupplz** [comment]
The instructions asked me not to give the EIP a number, hence some checks are failing

File: EIPS/eip-buy-russia-for-free-gas.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+eip: buy-russia-for-free-gas
+title: Buy Russia for free gas
+description: Ethereum Foundation should buy Russia to stop war on Ukraine and obtain virtually unlimited gas
+author: Nikita Kolmogorov (@backmeupplz)
+discussions-to: https://t.me/borodutcher
```
Comment (Pandapip1) [inline EIPS/eip-buy-russia-for-free-gas.md:6]: Should be on Ethereum magicians

File: EIPS/eip-buy-russia-for-free-gas.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+status: Draft
+type: Meta
+created: 2022-04-01
+---
+
+## Abstract
+Russia has a lot of gas. Also the Russia-Ukraine war. Gas prices are too damn high. I had to pay $60 for Gnosis Safe transaction yesterday. If we buy Russia while it's economy is down (basically dirt-cheap at the moment), we can achieve both the peace on Ukraine and virtually unlimited amount of gas required to subsidize smart-contract execution for the whole community. It's a win-win.
```
Comment (Pandapip1) [inline EIPS/eip-buy-russia-for-free-gas.md:13]:
```
Russia has a lot of gas.
```
[citation needed]

File: EIPS/eip-buy-russia-for-free-gas.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Ethereum Foundation MUST buy Russia and SHALL obtain gas for the community and REQUIRED to end war on Ukraine.
+
+## Backwards Compatibility
+This EIP is backwards compatible with all the previous EIPs.
+
+## Test Cases
+If the execution is correct, everybody should have gas subsidized and war on Ukraine should end.
+
+## Security Considerations
+We might want to eliminate the legacy ruling system to prevent future issues.
```
Comment (Pandapip1) [inline EIPS/eip-buy-russia-for-free-gas.md:28]: Needs reference to eip-225

File: EIPS/eip-buy-russia-for-free-gas.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (Pandapip1) [inline EIPS/eip-buy-russia-for-free-gas.md:31]: You should probably add that this eip SHALL be made publicly available in Russia.

File: EIPS/eip-buy-russia-for-free-gas.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+status: Draft
+type: Meta
+created: 2022-04-01
+---
+
+## Abstract
+Russia has a lot of gas. Also the Russia-Ukraine war. Gas prices are too damn high. I had to pay $60 for Gnosis Safe transaction yesterday. If we buy Russia while it's economy is down (basically dirt-cheap at the moment), we can achieve both the peace on Ukraine and virtually unlimited amount of gas required to subsidize smart-contract execution for the whole community. It's a win-win.
+
+## Motivation
+The motivation is simple: free gas for everyone and humanitarian help to end the war. Vitalik can them be nominated for the Nobel Peace Prize.
```
Comment (Pandapip1) [inline EIPS/eip-buy-russia-for-free-gas.md:16]:
```suggestion
The motivation is simple: free gas for everyone and humanitarian help to end the war. Vitalik can then be nominated for the Nobel Peace Prize.
```

**Review by Pandapip1 — COMMENTED** [review]
LGTM except for a few small issues

File: EIPS/eip-buy-russia-for-free-gas.md (hunk 4 above)
Comment (backmeupplz) [reply EIPS/eip-buy-russia-for-free-gas.md:31]: Done

File: EIPS/eip-buy-russia-for-free-gas.md (hunk 3 above)
Comment (backmeupplz) [reply EIPS/eip-buy-russia-for-free-gas.md]: Done

File: EIPS/eip-buy-russia-for-free-gas.md (hunk 2 above)
Comment (backmeupplz) [reply EIPS/eip-buy-russia-for-free-gas.md]: Added citation

File: EIPS/eip-buy-russia-for-free-gas.md (hunk 1 above)
Comment (backmeupplz) [reply EIPS/eip-buy-russia-for-free-gas.md:6]: Created a topic, but it has to be reviewed by a moderator. Will add the link to the Ethereum magicians as soon as the topic is up

**Comment by MicahZoltu** [comment]
I'm closing all PRs that were open on April 1st since we received a number of "joke" EIPs.  If this was in fact a serious EIP that you would like to have reviewed, please resubmit.

Note: Please keep in mind that EIP editors are all volunteers and have more work than time available.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Please fix the walidator errors

**Pull request closed without merging**